rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1990,9 +2017,7 @@ def pipe_operation(
         cmd = cmd + policy_detail
 
     utils.exec_shell_cmd(cmd)
-    if bucket_name is None:
-        update_commit()
-
+    update_commit()
```

Comment:
@viduship 
I think you are reverting changes merged in this PR by @anrao19 
https://github.com/red-hat-storage/ceph-qe-scripts/pull/556/files

could you please check..

Reply:
resolved, i think it got merged hence